.pre-commit-config.yaml

```diff
@@ -1,14 +1,14 @@
 repos:
   - repo: https://github.com/ambv/black
-    rev: 22.6.0
+    rev: 23.1.0
     hooks:
       - id: black
   - repo: https://github.com/pycqa/isort
-    rev: 5.6.3
+    rev: 5.12.0
     hooks:
       - id: isort
         name: isort
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: "v2.5.1"
+    rev: v2.7.1
     hooks:
```

Comment:
un-revert these bumps

Reply:
These were manually added. Was having some trouble with pre-commit for some reason.